core/rfb.js

```diff
@@ -243,15 +240,13 @@ export default function RFB(defaults) {
 
 RFB.prototype = {
     // Public methods
-    connect: function (host, port, creds, path) {
-        this._rfb_host = host;
-        this._rfb_port = port;
+    connect: function (url, creds) {
+        this._url = url;
         this._rfb_credentials = (creds !== undefined) ? creds : {};
-        this._rfb_path = (path !== undefined) ? path : "";
 
-        if (!this._rfb_host) {
-            return this._fail(
-                _("Must set host"));
+        if (!url) {
+            this._fail(_("Must specify URL"));
```

Comment:
👎 translation change :(

Reply:
Yes, that's unfortunate. But it's not a major issue for most users as we have a check in `ui.js` as well. So it is people using `RFB` directly that will be affected. And we haven't sorted out translations in that scenario.

Reply:
Ideally, yes. We could also have some compromise with codes covering broad areas and an explaining text similar to how the `Error` exception is used.

Do you know how browsers handle translation in those cases? Or are they always in English?

Reply:
Chrome does not have translations for warnings and errors that aren't intended for end-users. But I think we can be better, it's always a bit shitty to have mixed languages if you can avoid it.